BridgeSDK/BridgeSDK.m

```diff
@@ -40,6 +40,11 @@
 
 @implementation BridgeSDK
 
++ (void)setupWithBridgeInfo:(id <SBBBridgeInfoProtocol>)bridgeInfo {
+    gSBBPemCertificateName = bridgeInfo.certificateName;
```

Comment:
I'm actually going to be handling this a different way (copying/normalizing bridgeInfo values to a shared instance of a concrete class, then accessing the values via that shared instance throughout BridgeSDK instead of a bunch of separate globals). So no need to define/set this global--just ignore for now.